### ISSUE TITLE
I get this error when i try to use this on my app

### ISSUE THREAD

**Issue by fisforfaheem** [aagarwal1012/IntroViews-Flutter#80]
../../../Developer/flutter/.pub-cache/hosted/pub.dartlang.org/intro_views_flutter-2.8.2/lib/intro_views_flutter.dart:237:7: Error: No named parameter with the name 'resizeToAvoidBottomPadding'.
      resizeToAvoidBottomPadding: false,
      ^^^^^^^^^^^^^^^^^^^^^^^^^^
../../../Developer/flutter/packages/flutter/lib/src/material/scaffold.dart:1451:9: Context: Found this candidate, but the arguments don't match.

  const Scaffold({
        ^^^^^^^^

**Comment by Robert3141** [comment]
Yeah. I get the same error:
```
Launching lib\main.dart on Windows in debug mode...
lib\main.dart:1
/C:/tools/flutter/.pub-cache/git/IntroViews-Flutter-d54bf5670c6510a6c81f25970de3f174e46c8068/lib/intro_views_flutter.dart(237,7): error GBF4691A2: No named parameter with the name 'resizeToAvoidBottomPadding'. [C:\Users\robert\dev\cosmocannons\build\windows\flutter\flutter_assemble.vcxproj]
Exception: Build process failed.
```
I'm using landscape mode so this could be a bug with that?

**Comment by fisforfaheem** [comment]
no idea

**Comment by mzdm** [comment]
I believe this was fixed in the newest version (2.9.0) with https://github.com/aagarwal1012/IntroViews-Flutter/pull/83. If the issue still persists, please reopen the issue.

**Comment by fisforfaheem** [comment]
Ok make tutorial easy and a youtbe link
